Automattic/jetpack
# Widgets: PHPCS + Fusion Milestone widget

As part of WordPress.com's work to remove/reduce jQuery deps, we discovered that the Milestone widget was not being actively fusioned with WP.com and did not get the update to remove jQuery that Jetpack committed a few months back.

#### Changes proposed in this Pull Request:
* PHPCS Compliance in order to pass current WP.com linting.
* Update the version number of assets to be common to today on both platforms.
* Adds a class_exists before using the `Jetpack_AMP_Support` class since it is not presently available on WP.com.

#### Jetpack product discussion
None, but supports webperf's jQuery work.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Ensure Widgets are enabled on the Jetpack test site.
* Visit the Appearance->Widgets wp-admin page to ensure no errors. Use/edit the widget.
* View the front end of a site with a milestone widget. No errors in the console or PHP?

#### Proposed changelog entry for your changes:
* Can be included as part of a generic PHPCS/code improvement changelog entry.
